corehq/apps/smsforms/models.py

```diff
@@ -38,8 +41,40 @@ class SQLXFormsSession(models.Model):
     workflow = models.CharField(null=True, blank=True, max_length=20)
     reminder_id = models.CharField(null=True, blank=True, max_length=50)
 
+    # The number of minutes after which this session should expire, starting from the start_date.
+    expire_after = models.IntegerField(default=90 * 24 * 60)
+
+    # True if the session is still open.
+    session_is_open = models.BooleanField(default=True)
+
+    # A list of integers representing the intervals, in minutes, that reminders should be sent.
+    # A reminder in this context just sends the current question of an open survey to the contact
+    # in order to remind them to answer it. This can be empty list if no reminders are desired.
+    reminder_intervals = jsonfield.JSONField(default=list)
```

Comment:
Now that we are on Django 1.11, can we use the builtin Django JSONfield? https://docs.djangoproject.com/en/2.0/ref/contrib/postgres/fields/#django.contrib.postgres.fields.JSONField.

Reply:
updated: https://github.com/dimagi/commcare-hq/pull/19104/commits/da4c9969d7d647eba29935d4fc92852fff404257